docker-compose.yml

```diff
@@ -22,6 +22,11 @@ services:
       - 'SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_KEYCLOAK_CLIENTSECRET'
       - 'SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI'
       - 'SPRING_THYMELEAF_PREFIX=classpath:templates'
+      - 'TERRAWARE_ATLASSIAN_ACCOUNT'
+      - 'TERRAWARE_ATLASSIAN_APIHOST'
+      - 'TERRAWARE_ATLASSIAN_APITOKEN'
+      - 'TERRAWARE_ATLASSIAN_ENABLED'
+      - 'TERRAWARE_ATLASSIAN_SERVICEDESKKEY'
```

Comment:
Are these changes necessary?

Reply:
We need these for the BE support endpoints to work. I tested them so even when the env are not set, they will work correctly. I suspect we should add in the Dropbox or GDrive stubs here too.